### PR TITLE
[WIP] add DeleteVolumeGroupSnapshot API to delete group snapshot

### PR DESCRIPTION
**Pull request by yati1998**
Add Logic to DeleteVolumeGroupSnapshot API to delete VolumeGroupSnapshot

### CODE REVIEW

**Comment by xing-yang** [comment]
@yati1998 Can you fix the CLA issue?

**Comment by xing-yang** [comment]
Please add a release note.

File: pkg/group_snapshotter/group_snapshotter.go [hunk 1]
```diff
@@ -74,8 +75,21 @@ func (gs *groupSnapshot) CreateGroupSnapshot(ctx context.Context, groupSnapshotN
 
 }
 
-func (gs *groupSnapshot) DeleteGroupSnapshot(ctx context.Context, groupSnapshotID string, snapshotterCredentials map[string]string) error {
+func (gs *groupSnapshot) DeleteGroupSnapshot(ctx context.Context, groupSnapshotID string, snapshotIds []string, snapshotterCredentials map[string]string) error {
 	// TODO: Implement DeleteGroupSnapshot
```
Comment (RaunakShah) [inline pkg/group_snapshotter/group_snapshotter.go:79]: You can remove this TODO

File: pkg/sidecar-controller/csi_handler.go [hunk 2]
```diff
@@ -165,6 +167,30 @@ func (handler *csiHandler) CreateGroupSnapshot(content *crdv1alpha1.VolumeGroupS
...
+		return fmt.Errorf("cannot delete group snapshot. Snapshots to be deleted not found in group snapshot content %s", content.Name)
+	}
+	var groupSnapshotHandle string
+
+	if content.Status != nil && content.Status.VolumeGroupSnapshotHandle != nil {
+		groupSnapshotHandle = *content.Status.VolumeGroupSnapshotHandle
+	} else if content.Spec.Source.VolumeGroupSnapshotHandle != nil {
+		groupSnapshotHandle = *content.Spec.Source.VolumeGroupSnapshotHandle
+	} else {
+		return fmt.Errorf("failed to delete snapshot content %s: snapshotHandle is missing", content.Name)
```
Comment (RaunakShah) [inline pkg/sidecar-controller/csi_handler.go:188]: snapshot content %s: snapshotHandle  -> group snapshot content %s: groupSnapshotHandle

**Comment by xing-yang** [comment]
https://github.com/kubernetes-csi/external-snapshotter/pull/882 is merged.

**Pull request closed without merging**